velox/expression/VectorWriters.h

```diff
@@ -277,8 +277,8 @@ struct VectorWriter<Row<T...>> : public VectorWriterBase {
 
   void ensureSize(size_t size) override {
     if (size > rowVector_->size()) {
+      resizeVectorWriters<0>(size);
```

Comment:
can you add comment that we do that order in purpose and specify the purpose,

Reply:
maybe to ensure data_ is cached correctly .
make sure to run format on the comment, sapace missing.